app/views/admin/site_customization/pages/_form.html.erb

```diff
@@ -30,28 +31,30 @@
       <br>
     <% end %>
   </div>
-  <div class="small-12 medium-4 column">
-    <%= f.label :locale %>
-    <%= f.select :locale, I18n.available_locales.map { |l| [name_for_locale(l), l.to_s] }, { include_blank: true, label: false } %>
-  </div>
+
   <div class="small-12 medium-4 column">
     <%= f.label :slug %>
     <%= f.text_field :slug, label: false, size: 80, maxlength: 80 %>
   </div>
-
     <div class="small-12 column">
       <hr>
-
-      <%= f.label :title %>
-      <%= f.text_field :title, label: false %>
-
-      <%= f.label :subtitle %>
-      <%= f.text_field :subtitle, label: false, size: 80, maxlength: 80 %>
-
-      <div class="ckeditor margin-top">
-        <%= f.label :content %>
-        <%= f.cktext_area :content, label: false, cols: 80, rows: 10,
-                          ckeditor: { language: I18n.locale, toolbar: "admin" } %>
+        <%= f.translatable_text_field :title %>
+        <%= f.translatable_text_field :subtitle %>
+        <div class="ckeditor">
+          <% @page.globalize_locales.each do |locale| %>
+            <% globalize(locale) do %>
+              <%= content_tag :span , class: "js-globalize-attribute",
+                                data: { locale: locale },
+                                style: display_translation?(locale) do %>
+              <%= f.cktext_area "content_#{locale.to_s.downcase.underscore.to_sym}",
+                                class: "js-globalize-attribute",
+                                ckeditor: { language: locale },
+                                data: { locale: locale },
+                                label: false %>
```

Comment:
I think we need (somehow) a label here. Maybe you could use the code @microweb10 hast just implemented (still not merged, though): https://github.com/consul/consul/pull/2914/files#diff-4d169e27f1a8740fdcd328f352bf82d2R38

Reply:
Yes, our functionlity with ckeditor is similar. Will work just fine anything that @microweb10 implement. 👍